torchvision/csrc/cpu/vision_cpu.h

```diff
@@ -1,14 +1,24 @@
 #pragma once
 #include <torch/extension.h>
 
-std::tuple<at::Tensor, at::Tensor> ROIPool_forward_cpu(
+#ifdef _WIN32
+#if defined(torchvision_EXPORTS)
+#define VISION_API __declspec(dllexport)
+#else
+#define VISION_API __declspec(dllimport)
+#endif
+#else
+#define VISION_API
+#endif
```

Comment:
For the future, can you move those two to the same location in the root folder `csrc/` so that you don't need to implement it twice in the `cpu/` and `cuda/` folder?

Also. I wonder if it would be advisable to [re-use the macros from c10 in here?](https://github.com/pytorch/pytorch/blob/c193bd41f5eb3f09b8ff07f2a4ce39dcd19b01a7/c10/macros/Export.h#L84-L100)

Reply:
By inspecting the macros defined on C10, it would seem that they are the same, so we can use those